modules/cert-manager-configure-cloud-credentials-gcp-sts.adoc

```diff
@@ -0,0 +1,122 @@
+* The {cert-manager-operator} 1.11.1 or later is installed.
+
+.Procedure
+
+. Create a directory to store the `CredentialsRequest` custom resource (CR) by running the following command:
++
+[source,terminal]
+----
+$ mkdir credentials-request
+----
+
+. Change the directory by running the following command:
++
+[source,terminal]
+----
+$ cd credentials-request
+----
+
+. Create a `CredentialsRequest` resource YAML file, for example, `sample-credential-request.yaml`, as follows:
++
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  generation: 1
+  labels:
+    app: cert-manager
+    app.kubernetes.io/component: controller
+    app.kubernetes.io/instance: cert-manager
+    app.kubernetes.io/name: cert-manager
+  name: cert-manager
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: GCPProviderSpec
+    predefinedRoles:
+    - roles/dns.admin
+  secretRef:
+    name: gcp-credentials
+    namespace: cert-manager
+  serviceAccountNames:
+  - cert-manager
+----
+
+. Create a `CredentialsRequest` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sample-credential-request.yaml
+----
+
+. Use the `ccoctl` tool to process `CredentialsRequest` objects by running the following command:
++
+[source,terminal]
+----
+$ ccoctl gcp create-service-accounts \
+    --name <name> --output-dir=<path_to_output_dir> \
+    --credentials-requests-dir=<path_to_credrequests_dir> \
+    --workload-identity-pool <workload_identity_pool> \
+    --workload-identity-provider <workload_identity_provider> \
+    --project <project_name>
+----
+
+. Apply the secrets generated in the manifests directory of your cluster by running the following command:
++
+[source,terminal]
+----
+$ ls manifests/*-credentials.yaml | xargs -I{} oc apply -f {}
+----
+
+. Update the subscription object for {cert-manager-operator} by running the following command:
++
+[source,terminal]
+----
+$ oc -n cert-manager-operator patch subscription openshift-cert-manager-operator --type='merge' -p '{"spec":{"config":{"env":[{"name":"CLOUD_CREDENTIALS_SECRET_NAME","value":"gcp-credentials"}]}}}'
+----
+
+.Verification
+
+. Get the name of a cert-manager controller pod by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -l app.kubernetes.io/name=cert-manager -n cert-manager
+----
++
+.Example output
+[source,terminal]
+----
+NAME                          READY   STATUS    RESTARTS   AGE
+cert-manager-bd7fbb9fc-wvbbt  1/1     Running   0          15m39s
+----
+
+. Verify that GCP credentials are updated by running the following command:
++
+[source,terminal]
+----
+$ oc get -n cert-manager pod/<cert-manager_controller_pod_name> -o=jsonpath='{.spec.volumes}'
+----
++
+.Example output
+[source,terminal]
+----
+- name: cloud-credentials
+    secret:
+      secretName: gcp-creds
```

Comment:
You should mention the pod is updated with below GCP workload identity credential volumes mounted under below paths:
`oc get -n cert-manager pod/<cert-manager_controller_pod_name> -o yaml` example output:
```suggestion
spec:
  containers:
  - args:
    ...
    volumeMounts:
    - mountPath: /var/run/secrets/openshift/serviceaccount
      name: bound-sa-token
      ...
    - mountPath: /.config/gcloud
      name: cloud-credentials
  ...
  volumes:
  - name: bound-sa-token
    projected:
      ...
      sources:
      - serviceAccountToken:
          audience: openshift
          ...
          path: token
  - name: cloud-credentials
    secret:
      defaultMode: 420
      items:
      - key: service_account.json
        path: application_default_credentials.json
      secretName: cloud-credentials
```
https://issues.redhat.com/browse/OCPBUGS-9998?focusedId=22351582&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22351582 verification has details if needed for reference.